scss/_patterns_links.scss

```diff
@@ -1,7 +1,8 @@
 @import 'settings';
 
 @mixin vf-p-links {
-  .p-link {
+  .p-link,
+  a .p-link {
```

Comment:
Why do we need this? Is there an issue for it?

Reply:
Sorry, forgot to clean this up. I was testing what happens if you put the content of the entire card in an <a>, so it acts as one link and addressing things for that case. Removing as it should be in a separate issue if at all needed.